Package.swift

```diff
@@ -52,18 +53,32 @@ let package = Package(
       name: "SwiftInspectorAnalyzersTests",
       dependencies: [
         "SwiftInspectorAnalyzers",
+        "SwiftInspectorTestHelpers",
         "SwiftInspectorVisitors",
         "Nimble",
         "Quick",
     ], path: "Sources/SwiftInspectorAnalyzers/Tests"),
 
+    .target(
```

Comment:
Note that this target doesn't have an associated library declared at the top of this package. I believe that means we aren't exposing this library externally.

Reply:
Yup, that's correct

> Library
>
> A library product is used to vend library targets containing the public APIs that will be available to clients.

https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html